src/locale/locales/pt-BR/messages.po

```diff
@@ -8165,7 +8165,7 @@ msgstr "Você não silenciou nenhuma palavra ou tag ainda"
 #: src/components/moderation/ModerationDetailsDialog.tsx:117
 #: src/lib/moderation/useModerationCauseDescription.ts:125
 msgid "You hid this reply."
-msgstr ""
+msgstr "Você escondeu esta resposta."
```

Comment:
maybe change to "ocultou" since that has been used for translating "hide"

Reply:
ok